src/screens/ExplainSheet.js

```diff
@@ -536,8 +536,12 @@ export const explainers = (params, colors) => ({
   swapResetInputs: {
     button: {
       label: `Continue with ${networkInfo[params?.network]?.name}`,
-      bgColor: colors?.alpha(colors?.networkColors[params?.network], 0.06),
-      textColor: colors?.networkColors[params?.network],
+      bgColor:
+        colors?.networkColors[params?.network] &&
+        colors?.alpha(colors?.networkColors[params?.network], 0.06),
+      textColor:
```

Comment:
I have a feeling the second color doesn't need the checks. Seems like the failing part was again the chroma js

Reply:
yeah im gonna play with adding some safeties to that fn later